src/types.ts

```diff
@@ -17,3 +16,20 @@ export interface CodeInputValue {
   language?: string
   highlightedLines?: number[]
 }
+/**
+ * @public
+ */
+export interface CodeOptions {
+  theme?: string
+  darkTheme?: string
+  languageAlternatives?: CodeInputLanguage[]
+  language?: string
+  withFilename?: boolean
+}
+
+/**
+ * @public
+ */
+export type CodeSchemaType = Omit<ObjectSchemaType, 'options'> & {
```

Comment:
Prefer `interface` (re: guidelines)

Reply:
This is a guideline I personally subscribe to, but I often end up doing this when making composite types for whatever reason.